opendcs-rest-api/src/main/resources/swaggerui/open_api.json

```diff
@@ -7,10 +7,7 @@
   },
   "servers": [
     {
-      "url": ""
-    },
-    {
-      "url": "odcsapi"
+      "url": "."
```

Comment:
Maybe this will solve my issue with the OWASP Zap. Previously it was trying to make all of the endpoints relative to the swagger.json. I had to remove the empty URL, but maybe this will work better. Let's see what the GitHub Actions do once you rebase/merge with main